library/core/src/primitive_docs.rs

```diff
@@ -1317,6 +1317,15 @@ mod prim_f16 {}
 /// | `wasm32`, `wasm64` | If all input NaNs are quiet with all-zero payload: None.<br> Otherwise: all possible payloads. |
 ///
 /// For targets not in this table, all payloads are possible.
+///
+/// # Algebraic operators
+///
+/// Algebraic operators of the form `a.algebraic_*(b)` allow the compiler to reassociate individual
+/// floating point operations for better vectorization. Because of the unpredictable nature of
```

Comment:
It would be good to give the user an idea of what to expect. Maybe between these two sentences add something like:

> The exact set of optimizations is unspecified but typically allows combining operations, rearranging series of operations based on mathematical properties, converting between division and reciprocal multiplication, and disregarding the sign of zero. This may mean that the results of elementary operations have undefined precision, but will not invoke input-dependent undefined behavior in performing these operations.

And then move "Because ..." to a new paragraph.

Reply:
Added minus redundant "in performing these operations". How's that look?

Reply:
Looks good, thanks!